metrics/web/test/base/presentation/widgets/value_form_field_test.dart

```diff
@@ -80,7 +80,7 @@ class _ValueFormFieldTestbed<T> extends StatelessWidget {
 
   /// Defines whether the [ValueFormField] should validate
   /// the value each time it changes.
-  final bool autovalidate;
+  final AutovalidateMode autovalidateMode;
```

Comment:
Should we update the documentation of this field to be `An auto validation mode of the [ValueFormField].` or something like this?